### DISCUSSION TITLE
My Account got suspended for no reason and it is causing me trouble

### DISCUSSION THREAD

**Discussion by j-naman1506** [community/community#123236] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I have created multiple threads and tickets but there is no response from the support team. I am working in a company and the company linked mail address's github account got suspended. This is stopping the development process and my other works are stuck. So please help me in this.

Ticket ID: 2709776

**Comment by ebndev** [comment]
Hi @j-naman1506 ,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
